README.md

```diff
@@ -1,197 +1,45 @@
 # Datamaintain
 [![CircleCI](https://circleci.com/gh/4sh/datamaintain.svg?style=shield)](https://circleci.com/gh/4sh/datamaintain) ![GitHub](https://img.shields.io/github/license/4sh/datamaintain)
 
-Datamaintain is a Kotlin library that runs your scripts on your database and tracks the scripts runned. You may integrate it directly in your Java or Kotlin server or you may use the CLI.
+Your colleague added a column in the database without telling it to you and now nothing works ? Classic. To avoid this, use Datamaintain! This Kotlin library  helps maintaining consistency between data and code : it runs the new scripts added and tracks the scripts runned. According to your needs you may :
+- [Use it as a dependancy in your Java or Kotlin server](docs/asdependancy.md) : really handy to collaborate,
+- [And/or use the Command Line Interface (CLI)](docs/withcli.md) : enables more control when deploying.
```

Comment:
I would separate words with a dash in the file name. So `with-cli.md`. I find it more readable.
You already do it on many files, so it would also be consistent.